Sources/Printing/Print.swift

```diff
@@ -34,8 +34,8 @@ public func printBreadcrumb(_ items: Any...) -> String? {
  - Returns : text to be printed
  */
 @discardableResult
-public func printError(_ items: Any...) -> String? {
-    guard Printer.level != .nothing || Printer.level == .error else {
+public func Outputror(_ items: Any...) -> String? {
```

Comment:
Zeer rare rename naar die Output die verkeerd is gegaan. Snap het slaagt op niets wat hier staat. Opgelost in de volgende commit.